lnprototest/runner.py

```diff
@@ -120,6 +121,19 @@ def teardown(self):
         and it is used to clean up the root dir where the tests are run."""
         shutil.rmtree(self.directory)
 
+    def runner_features(
+        self,
+        additional_features: Optional[List[int]] = None,
+        field_name: Optional[str] = None,
+    ) -> str:
+        """
+        Provide the features required by the node.
+        """
+        if additional_features is None:
+            return ""
+        else:
+            return bitfield(*additional_features)
+
```

Comment:
```suggestion
    def runner_features(
        self,
        additional_features: Optional[List[int]] = None,
        globals: bool = False,
    ) -> str:
        """
        Provide the features required by the node.
        """
        if additional_features is None:
            return ""
        else:
            return bitfield(*additional_features)

```

Thinking that maybe a bool is better here? easier instead of a string that is error-prone to typos!

Reply:
Yeah having a bool would be better here. I will implement this.